Source/Charts/Components/MarkerView.swift

```diff
@@ -72,10 +72,15 @@ open class MarkerView: NSUIView, IMarker
     }
     
     @objc
-    open class func viewFromXib() -> MarkerView?
+    open class func viewFromXib() -> MarkerView? {
+        return viewFromXib(in: Bundle.main)
+    }
+    
+    @objc
+    open class func viewFromXib(in bundle: Bundle) -> MarkerView?
```

Comment:
This can be one method: `open class func viewFromXib(in bundle: Bundle = .main) -> MarkerView?` instead of two.

Reply:
Yeah that's what I was thinking in the first place, however if someone is overriding that method it will break it's code. (@objc) is declared as well here. So even more important!

What do you think?

Reply:
The main reason to override this method is to change the bundle. If it breaks their code, this is a very minor change and we will have a few possible minor code breaking changes in 3.1.

Let's make this one method.

Reply:
And keep the `{` on a new line.

Reply:
All done. Thanks. Great job by the way :)